api/src/docker/openscad/readme.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I'll delete this readme, made redundant by the better one in the api/src/docker. TODO